css/resume.css

```diff
@@ -0,0 +1,70 @@
+.title-res {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.normal-row {
+  height: 100px;
+}
+
+.narrow-row {
+  height: 50px;
+}
```

Comment:
You want your content to dictate how high elements are, not give each row a fixed height.
You can do all this with top and bottom margin and padding: you don't need separate rows for each separation between the white blocks, just add `margin-bottom` to each white block. 

Your blocks themselves should use `col-md-8 offset-md-2` to give them their width and center them. 

Then, inside each white block, you only need **1** row, and the content inside can be split by using `col-12` whenever you want to use the whole width.